database/data.sql

```diff
@@ -0,0 +1,2 @@
+insert into "chefs" ("name", "photoUrl", "bio")
+values ('Gordon Ramsay', 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ1KeAGmbQikVtmQ5bHa3_-5tGV9rp5DCifHQ&usqp=CAU', 'testing')
```

Comment:
I recommend downloading the picture to your `public/images` and then storing the url as `/images/ramsay.png`